index.php

```diff
@@ -369,14 +369,15 @@
         </section>
         <span id="translate-download" style="display:none;" hidden>Download elementary OS</span>
         <span id="translate-purchase" style="display:none;" hidden>Purchase elementary OS</span>
-        <div id="download-modal" class="modal">
-            <h3>Choose a Download</h3>
-            <p>For help and more info, see the <a class="read-more" href="docs/installation" target="_blank">installation guide</a></p>
-            <div class="row actions">
-                <div class="column">
-                    <a class="button close-modal" style="margin:0;" href="#">Cancel</a>
-                </div>
-                <div class="column linked">
+        <div id="download-modal" class="dialog">
+            <img alt="Download elementary OS icon" src="images/icons/apps/48/ubiquity.svg">
+            <div class="content-area">
+                <p class="primary">Choose a Download</p>
+                <p>Download from a localized server or by magnet link. For help and more info, see the <a class="read-more" href="docs/installation" target="_blank">installation guide</a></p>
+            </div>
+            <div class="action-area">
+                <a class="button close-modal" style="margin:0;" href="#">Cancel</a>
```

Comment:
Because this has `href="#"`, it scrolls the page to the top. Leave it blank and add the class `clickable` to cure.